astroid/builder.py

```diff
@@ -466,11 +468,13 @@ def _extract_single_node(code: str, module_name: str = "") -> nodes.NodeNG:
 
 
 def _parse_string(
-    data: str, type_comments: bool = True
+    data: str, type_comments: bool = True, modname: str | None = None
```

Comment:
```suggestion
    data: str, type_comments: bool = True, modname: str | None
```

Why not make it required?

Reply:
Non-default arguments can't follow ones with default.

Reply:
My bad!